napi_from_js.md

```diff
@@ -0,0 +1,204 @@
+# N-API Function APIs 
+
+N-API provides a set of APIs that allow JavaScript code to 
+call back into native code. N-API APIs that support calling back
+into JavaScript take in callback function represented by 
+the `napi_callback` type. When the JavaScript VM calls back to
+native code, the `napi_callback` provided is invoked. The APIs 
+documented in this section allow the callback function to do the 
+following:
+- Get information about the context in which the callback was invoked
+- Get the arguments passed into the callback
+- Return a `napi_value` back from the callback
```

Comment:
Either this doc or https://github.com/nodejs/abi-stable-node/pull/215 is missing coverage of `napi_create_function()`